src/utils/Chart.js

```diff
@@ -0,0 +1,35 @@
+const d3 = require('d3');
+
+const Chart = {
+  getDataMinMaxValues (data, axis) {
+    const max = d3.max(data, d => {
+      return Math.ceil(d[axis] / 1000) * 1000;
```

Comment:
Valid point @jmophoto.  Let me play with the math some more and see if I can make the `1000` more dynamic.